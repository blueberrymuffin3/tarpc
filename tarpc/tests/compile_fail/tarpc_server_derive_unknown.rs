#[tarpc::service(derive_serde = false, derive = "IDontExist")]
trait World {
    async fn hello(name: String) -> String;
}

fn main() {}
