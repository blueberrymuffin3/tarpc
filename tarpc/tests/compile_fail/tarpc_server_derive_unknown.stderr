error: cannot find derive macro `IDontExist` in this scope
 --> tests/compile_fail/tarpc_server_derive_unknown.rs:1:49
  |
1 | #[tarpc::service(derive_serde = false, derive = "IDontExist")]
  |                                                 ^^^^^^^^^^^^
